agent/client/client.go

```diff
@@ -748,6 +773,20 @@ func (c *Client) GetServerConnectMetadata() *agentpb.ServerConnectMetadata {
 	return md
 }
 
+func (c *Client) GetConnectedUpTime() float32 {
```

Comment:
how much memory and cpu it consumes?

Reply:
good question

do you know how can I measure that?

Reply:
You can profile or write perf test. Pls, see my other comments.